### PR TITLE
Statistics

### PR DESCRIPTION
**Pull request by mkorob**
image
image
 
Example of visualization

### CODE REVIEW

**Comment by mkorob** [comment]
image

File: src/components/views/game/Game.js [hunk 1]
```diff
@@ -171,6 +184,148 @@ const Game = ({id}) => {
         history.push('/home');
     }
 
+    /*
+    const convertTurnData_old = (statisticsData) => {
```
Comment (apple00juice) [inline src/components/views/game/Game.js:188]: Can that be removed?

**Review by apple00juice — APPROVED** [review]
Looks really good! 👍 
Just one question, how hard would it be to change the units killed per turn to a bar char? I think that would look better...

**Review by luis-tm — COMMENTED** [review]
Looks good!

Only one thing: regarding the naming of the players. We need to consider the actual names of the players in the game, don't we? (instead of Player-0, Player-1, etc).

**Review by luis-tm — APPROVED** [review]
Super!

**Pull request merged**